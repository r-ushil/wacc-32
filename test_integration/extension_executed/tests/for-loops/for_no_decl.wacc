begin

    int i = 0;

    for ; i < 5; i = i + 1 do
        print "hello"
    done ;

    exit i

end