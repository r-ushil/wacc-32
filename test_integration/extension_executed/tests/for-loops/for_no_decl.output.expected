Compiler exit code: 0
--------- stdout ---------
hellohellohellohellohello--------------------------
Exit code: 5
